js/react/components/auth/login/LoginWithEmail.jsx

```diff
@@ -1,72 +1,141 @@
 import PropTypes from 'prop-types';
+import { useState } from 'react';
 
 import { SIGNUP } from '../../../utils/constants.jsx';
+import CustomToast from '../../shared/CustomToast.jsx';
 import Modal from '../../shared/Modal.jsx';
 
 function LoginWithEmail({ modalRef, dispatch }) {
+	const [email, setEmail] = useState('');
+	const [password, setPassword] = useState('');
+	const [error, setError] = useState('');
+	const [hidePassword, setHidePassword] = useState(true);
+	const togglePasswordVisibility = () => {
+		setHidePassword((prev) => !prev);
+	};
+
+	const loginWithEmailHandler = async (e) => {
+		e.preventDefault();
+		if (email === '' || password === '') {
+			setError('Please fill out email and password');
+			setTimeout(() => {
+				setError('');
+			}, 5000);
+			return;
+		}
+		if (!email.split('').includes('@')) {
+			setError('Please provide valid email');
+			setTimeout(() => {
+				setError('');
+			}, 5000);
+			return;
+		}
+		// TBD get base url from env local
+		const response = await fetch('http://localhost:3000/login', {
+			method: 'POST',
+			headers: {
+				Accept: 'application/json',
+				'Content-Type': 'application/json',
+			},
+			body: JSON.stringify({ email, password }),
+		});
+		if (response.ok) {
+			const data = await response.json();
+			return data;
+		}
+		// console.log(response);
+		setEmail('');
```

Comment:
```suggestion
		}
		
		setEmail('');
```

removed unused console.log